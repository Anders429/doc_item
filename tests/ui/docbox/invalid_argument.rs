use doc_item::docbox;

#[docbox(invalid="argument")]
fn invalid_arguments() {}

fn main() {}
