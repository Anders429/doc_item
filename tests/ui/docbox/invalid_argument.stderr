error: Unknown field: `invalid`
 --> $DIR/invalid_argument.rs:3:10
  |
3 | #[docbox(invalid="argument")]
  |          ^^^^^^^^^^^^^^^^^^
